crates/rattler_package_streaming/src/reqwest/mod.rs

```diff
@@ -3,11 +3,17 @@
 #[cfg(feature = "tokio")]
 pub mod tokio;
 
+#[cfg(feature = "blocking")]
```

Comment:
Perhaps it would be better to move all the blocking things to a module to remove the code duplication of the blocking feature cfg?